src/main/java/core/basesyntax/Draw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Draw {
+    public abstract String drawing();
+}
```

Comment:
Interface names are better to be nouns or adjectives
Methods names are better to be verbs
```suggestion
public interface Drawable {
    public abstract String draw();
}
```